packages/cipherstash-proxy/src/config/tls.rs

```diff
@@ -0,0 +1,195 @@
+use std::path::PathBuf;
+
+use rustls_pki_types::{pem::PemObject, CertificateDer, PrivateKeyDer};
+use serde::Deserialize;
+use tracing::debug;
+
+use crate::log::CONFIG;
+
+///
+/// Server TLS Configuration
+/// This is listener/inbound connection config
+///
+#[derive(Clone, Debug, Deserialize)]
+#[serde(tag = "type")]
+pub enum TlsConfig {
+    Pem {
+        certificate: String,
+        private_key: String,
```

Comment:
Not now necessarily but this would be a good candidate for VitaminC Protected as well.
Either that or we perhaps consider using Zeroize directly.